pkg/util/common/common_test.go

```diff
@@ -93,7 +94,9 @@ func TestStructToMap(t *testing.T) {
 		},
 	}
 
-	assert.Equal(t, map[string]interface{}{
+	var topNil Top
+
+	topOutput := map[string]interface{}{
```

Comment:
Could you please inline these outputs into the table below, it will be easier to follow that way.

Reply:
will do, thanks